### PR TITLE
[query] Lower TableMapRows with scans

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: hail/src/main/scala/is/hail/expr/ir/Emit.scala [hunk 1]
```diff
@@ -877,7 +877,7 @@ class Emit[C](
           cb += srvb.advance()
         }, "aggs_result")
 
-        sc.store(cb)
+//        sc.store(cb)
```
Comment (catoverdrive) [inline hail/src/main/scala/is/hail/expr/ir/Emit.scala:880]: Why do we need to take this out? Store has logic in it that releases region references for aggregator state fields.

Comment (tpoterba) [reply hail/src/main/scala/is/hail/expr/ir/Emit.scala:880]: ResultOP can be called multiple times. If it doesn't load, it can't store.

Comment (catoverdrive) [reply hail/src/main/scala/is/hail/expr/ir/Emit.scala:880]: Womp. Do we have a thing to clean up region references post-aggregation/can we guarantee that they're cleared if we take this out?

Comment (tpoterba) [reply hail/src/main/scala/is/hail/expr/ir/Emit.scala:880]: I'll investigate this...

Comment (tpoterba) [reply hail/src/main/scala/is/hail/expr/ir/Emit.scala:880]: Kinda think RunAgg should clean up after

**Pull request merged**